index.Rmd

```diff
@@ -13,14 +13,14 @@ editor_options:
 # Introduction
 
 
-The deer tick or blacklegged tick (<i>Ixodes scapularis</i>) is the most medically relevant species of hard tick that in New York State (NYS). <i>I. scapularis</i> populations in NYS may carry a plethora of pathogens, including but not limited to:
+The deer tick or blacklegged tick (<i>Ixodes scapularis</i>) is the most medically relevant species of hard tick that in New York State (NYS). <i>I. scapularis</i> populations in NYS may carry a plethora of pathogens, including but not limited to: 
```

Comment:
I think Adam has suggested this to you before, so ignore this if you already know, but you can use underscores to italicize a word instead of <i>, so you can do _word_. Whatever works for you is fine, but I thought I'd at least say something.